### PR TITLE
HAWQ-1312. Forbid grant/revoke command in HAWQ side once Ranger is co…

### PR DESCRIPTION
**Pull request by linwen**
HAWQ-1312. Forbid grant/revoke command in HAWQ side once Ranger is configured.

### CODE REVIEW

File: src/backend/catalog/aclchk.c [hunk 1]
```diff
@@ -298,6 +298,14 @@ ExecuteGrantStmt(GrantStmt *stmt)
...
 
 	/*
+	 * Don't allow GRANT/REVOKE if in ranger mode.
+	 */
+	if (aclType == HAWQ_ACL_RANGER)
+	{
+		elog(ERROR, "GRANT/REVOKE is not allowed in ranger mode");
+	}
+
+	/*
```
Comment (ictmalili) [inline src/backend/catalog/aclchk.c]: I don't think we should forbid all the objects for grant/revoke. Given that we decide to leave catalog table/owner check in HAWQ side, we should allow grant/revoke for these operations.  What we need to forbid is what we have supported in Ranger UI configuration.

Comment (linwen) [inline src/backend/catalog/aclchk.c]: Thank you for pointing out!

**Comment by zhangh43** [comment]
+1

**Comment by interma** [comment]
+1

**Comment by stanlyxiang** [comment]
+1

**Pull request closed without merging**